src/api/spec/controllers/webui/watched_items_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe Webui::WatchedItemsController, type: :controller do
+  describe '#toggle' do
+    context 'when the user belongs to the beta group' do
+      before do
+        Flipper.enable(:new_watchlist, user)
+      end
+
+      context 'when the watched item is not already watched' do
+        let(:user) { create(:confirmed_user) }
+        let(:package) { create(:package) }
+
+        before do
+          login user
+          put :toggle, params: { package: package, project: package.project }
+        end
+
+        it 'redirects back' do
```

Comment:
Why test the redirect? You expect `ActionController::Redirecting ` to break? :)

Reply:
Okay, we'll remove this kind of tests, they don't add much value...

Reply:
done